DOC-2271 doc(cdc): change CDC message MAP attribute formati; DOC-2031 doc(cdc): add graph field to CDC message [3.11] (#532)
tigergraph/server-docs: modules/system-management/pages/change-data-capture/cdc-message-example.adoc

“*Transaction*: epoch time of transaction commit message.”

¦ `graph`

> ¦ String: ”name of graph updated type belongs to. For global type this is `"GLOBAL"`”.

Reviewer: ```suggestion
¦ String: ”GLOBAL", "<local_graph_name>"
```

}`
----

> [NOTE]

Reviewer: ```suggestion
[NOTE]
.Content format change
```

====

> Format for `"value"` of MAP attribute has been updated to ensure consistency with the xref:tigergraph-server:API:index.adoc#_formatting_data_in_json[POST request data format] in TigerGraph v3.11.0, v4.1.1, v4.2.0 and subsequent releases. For instance:

Reviewer: ```suggestion
Beginning with version 3.11, the format for the value of the `content` message field has been revised to be consistent with the xref:tigergraph-server:API:index.adoc#_formatting_data_in_json[POST request data format].
For example:
```

* Updated format: `{ "keylist":["k1", "k2"], "valuelist": [10.5, 24.8] }`
* Legacy format: `{ "k1": 10.5, "k2": 24.8 }`

> The lagacy format is utilized only if adding `CDCLegacyFormat=true;` in `gadmin config entry GPE.BasicConfig.Env`.

Reviewer: ```suggestion
The legacy format is still available by setting `CDCLegacyFormat=true;` in `gadmin config entry GPE.BasicConfig.Env`.
```
